code/modules/clothing/head/hood.dm

```diff
@@ -10,42 +10,159 @@
 /obj/item/clothing/head/hood/winter
 	name = "winter hood"
 	desc = "A hood attached to a heavy winter jacket."
-	icon_state = "generic_hood"
+	icon_state = "winterhood"
 	min_cold_protection_temperature = SPACE_SUIT_MIN_COLD_PROTECTION_TEMPERATURE
 
 /obj/item/clothing/head/hood/winter/captain
 	name = "site manager's winter hood"
+	desc = "A blue and yellow hood attached to a heavy winter jacket."
+	icon_state = "winterhood_captain"
 	armor = list(melee = 20, bullet = 15, laser = 20, energy = 10, bomb = 15, bio = 0, rad = 0)
 
+/obj/item/clothing/head/hood/winter/hop
+	name = "head of personnel's winter hood"
+	desc = "A cozy winter hood attached to a heavy winter jacket."
+	icon_state = "winterhood_hop"
+
 /obj/item/clothing/head/hood/winter/security
 	name = "security winter hood"
+	desc = "A red, armor-padded winter hood."
+	icon_state = "winterhood_security"
+	armor = list(melee = 25, bullet = 20, laser = 20, energy = 15, bomb = 20, bio = 0, rad = 0)
+
+/obj/item/clothing/head/hood/winter/security/hos
+	name = "head of security's winter hood"
+	desc = "A red, armor-padded winter hood, lovingly woven with a Kevlar interleave. Definitely not bulletproof, especially not the part where your face goes."
+	icon_state = "winterhood_hos"
 	armor = list(melee = 25, bullet = 20, laser = 20, energy = 15, bomb = 20, bio = 0, rad = 0)
 
 /obj/item/clothing/head/hood/winter/medical
 	name = "medical winter hood"
+	desc = "A white winter coat hood."
+	icon_state = "winterhood_medical"
+	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 50, rad = 0)
+
+/obj/item/clothing/head/hood/winter/medical/alt
+	name = "medical winter hood, alt"
+	desc = "A white winter coat hood. It's warm."
+	icon_state = "winterhood_medicalalt"
+	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 50, rad = 0)
```

Comment:
If the armor value is the same as defined on the base type, it doesn't need to be redefined as the same value on all subtypes.